packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -168,13 +169,20 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
                 </h4>
                 {showDummyData && <DummyDataInput setValues={props.setValues} />}
                 <p className={styles.label01}>{t('jumpTo', 'Jump to')}</p>
-                {sections.map((section) => (
-                  <div className={`${styles.space05} ${styles.touchTarget}`} key={section.name}>
-                    <Link className={styles.linkName} onClick={() => scrollIntoView(section.id)}>
-                      <XAxis size={16} /> {t(`${section.id}Section`, section.name)}
-                    </Link>
-                  </div>
-                ))}
+                <div className={styles.linkContainer}>
+                  {sections.map((section) => (
+                    <div className={`${styles.space05} ${styles.touchTarget}`} key={section.name}>
+                      <Link
+                        className={`${styles.linkName} ${activeLink === section.id && styles.activeLink}`}
+                        onClick={() => {
+                          setActiveLink(section.id);
```

Comment:
Updating active link state to add the active link class

Reply:
Why using an activeLink?

Reason being that user can scroll from one section to another and then this thing is also to be handled, else the user scrolls to another section and the activeLink will be highlighted to the prev section only. 

I suggest not to take this ahead.
Thanks!

Reply:
Removed. Thanks